framework/src/play/i18n/Messages.java

```diff
@@ -32,9 +32,9 @@
 
     private static final Object[] NO_ARGS = new Object[] { null };
 
-    static public Properties defaults;
+    public static Properties defaults = new Properties();
 
-    static public Map<String, Properties> locales = new HashMap<>();
+    public static Map<String, Properties> locales = new HashMap<>();
```

Comment:
@theo 
* `defaults` is re-initialized in `MessagesPlugin` and many tests - so it cannot be final. 
* I did `locales` final in branch 1.4.x